cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/Select/Select.tsx

```diff
@@ -0,0 +1,61 @@
+// React imports
+import React, { ChangeEvent } from 'react';
+
+// Library imports
+import { CdsSelect } from '@cds/react/select';
+import { CdsControlMessage } from '@cds/react/forms';
+import { CdsIcon } from '@cds/react/icon';
+import { UseFormRegister } from 'react-hook-form';
+import { CdsProgressCircle } from '@cds/react/progress-circle';
+
+// App imports
+import './Select.scss';
+
+interface Props {
+    label?: string;
+    className?: string;
+    defaultValue?: string;
+    error?: string;
+    disabled?: boolean;
+    controlMessage?: string;
+    isLoading?: boolean;
+    name: string;
+    handleSelect: (event: ChangeEvent<HTMLSelectElement>) => void;
+    children: React.ReactNode;
+    register: UseFormRegister<any>;
+}
+function Select(props: Props) {
+    const { children, controlMessage, className, defaultValue, disabled, error, handleSelect, label, name, register, isLoading } = props;
+    return (
+        <div className="select-container">
+            <CdsSelect layout="compact">
+                <label>
+                    {label} <CdsIcon shape="info-circle" size="md"></CdsIcon>
```

Comment:
Since this info-circle icon is for the tooltip, I think we should pass the tooltip content in through the props optionally and only display the icon if the tooltip content is present.

Reply:
added